doc/chapters/gateways/index.rst

```diff
@@ -302,12 +302,39 @@ An example configuration can look like this::
 Network gateway
 ===============
 
-The Network Gateway is used to provide Internet access through a container-specific firewall.
+The Network Gateway is used to setup network connection and configure which traffic is allowed and not.
 
 Configuration
 -------------
+Types: "INCOMMING" and "OUTGOING"
+Targets: "ACCEPT", "DROP" and "REJECT"
+host: hostname or ip-address with netmask
+default: target to choose when not matching any other rule
 
-T.B.D.
+Example of network gateway config::
+
+    [
+        {
+            "type": "OUTGOING",
+            "rules": [
+                         { "host": "127.0.0.1/16", "port": 80, "target": "ACCEPT"},
+                         { "host": "google.com", "port": "80-85", "target": "ACCEPT"},
+                         { "host": "127.0.0.1/16", "port": [80, 8080], "target": "ACCEPT"},
+                         { "host": "50.63.202.33/24", "target": "DROP"},
+                     ],
+            "default": "REJECT"
+        },
+        {
+            "type": "INCOMMING",
+            "rules": [
+                         { "host": "127.0.0.1/16", "port": 80, "target": "ACCEPT"},
+                         { "host": "google.com", "port": "80-85", "target": "ACCEPT"},
+                         { "host": "127.0.0.1/16", "port": [80, 8080], "target": "ACCEPT"},
+                         { "host": "50.63.202.33/24", "target": "REJECT"},
+                     ],
+            "default": "DROP"
+        }
+    ]
```

Comment:
Actually empty lists are allowed but not specifying the rules parameter in __not__ allowed. Which means that this is not an issue.